xml/decoders/TCS_WOW_steam046.xml

```diff
@@ -0,0 +1,735 @@
+         </variable>
+         <variable item="Sound Group 7 Checkbox 1" CV="T2CV.4.19" default="1" mask="XXXXXXXXXXXXXXXV">
+            <xi:include href="http://jmri.org/xml/decoders/parts/enum-OnOff.xml"/>
+            <label>Chuff Cam</label>
+         </variable>
+         <variable item="Sound Group 7 Checkbox 3" CV="T2CV.4.19" default="0" mask="XXXXXXXXXXXXXVXX">
+            <xi:include href="http://jmri.org/xml/decoders/parts/enum-OnOff.xml"/>
+            <label>Mute on Startup</label>
+         </variable>
+		 <variable item="Sound Group 7 Checkbox 5" CV="T2CV.4.19" default="1" mask="XXXXXXXXXXXVXXXX">
+            <xi:include href="http://jmri.org/xml/decoders/parts/enum-OnOff.xml"/>
+            <label>Rotate Whistle Auditory Feedback</label>
+         </variable>
+		 		 <variable item="Sound Group 7 Checkbox 6" CV="T2CV.4.19" default="1" mask="XXXXXXXXXXVXXXXX">
+            <xi:include href="http://jmri.org/xml/decoders/parts/enum-OnOff.xml"/>
+            <label>Light Mode and Sound Mode Auditory Feedback</label>
+         </variable>
+         <variable item="Sound Group 7 Checkbox 8" CV="T2CV.4.19" default="1" mask="XXXXXXXXVXXXXXXX">
+            <xi:include href="http://jmri.org/xml/decoders/parts/enum-OnOff.xml"/>
+            <label>Function 2 is Momentary (improves Audio Assist)</label>
+         </variable>
+         <variable item="Sound Option 3" CV="T2CV.4.19" default="0" mask="XXXXXXXVXXXXXXXX">
+            <enumVal>
+               <enumChoice choice="Non Articulated"/>
+               <enumChoice choice="Articulated"/>
+            </enumVal>
+            <label>Articulation type</label>
+         </variable>
+         <variable item="Sound Group 8 Checkbox 3" CV="T2CV.4.19" default="0" mask="XXXXXVXXXXXXXXXX">
+            <xi:include href="http://jmri.org/xml/decoders/parts/enum-OnOff.xml"/>
+            <label>Reverse Directional whistles</label>
+         </variable>
+         <variable item="Sound Group 8 Checkbox 4" CV="T2CV.4.19" default="1" mask="XXXXVXXXXXXXXXXX">
+            <xi:include href="http://jmri.org/xml/decoders/parts/enum-OnOff.xml"/>
+            <label>Power Braking Enabled</label>
+         </variable>
+         <variable item="Sound Group 8 Checkbox 5" CV="T2CV.4.19" default="1" mask="XXXVXXXXXXXXXXXX">
+            <xi:include href="http://jmri.org/xml/decoders/parts/enum-OnOff.xml"/>
+            <label>Switching and Mainline momentum Feedback</label>
+         </variable>
+         <variable item="Sound Group 8 Checkbox 6" CV="T2CV.4.19" default="1" mask="XXVXXXXXXXXXXXXX">
+            <xi:include href="http://jmri.org/xml/decoders/parts/enum-OnOff.xml"/>
+            <label>Johnson Bar Feedback</label>
+         </variable>
+         <variable item="Sound Group 8 Checkbox 7" CV="T2CV.4.19" default="1" mask="XVXXXXXXXXXXXXXX" toolTip="Enabling this setting will allow all 28 functions to be controlled using only the first ten buttons on the throttle">
```

Comment:
Windows CI fails on this line:
`Attribute 'toolTip' is not allowed to appear in element 'variable'`

This probably applies to line 399 as well.

For the solution, see line 484:
`<tooltip>If checked, each secessive press of quick button with play random quill sound, unchecked will play in order</tooltip>`